test/cppsuite/src/main/validator.cpp

```diff
@@ -26,7 +26,7 @@
  * OTHER DEALINGS IN THE SOFTWARE.
```

Comment:
I'd prefer validation.h over validator.h, but open to discussing this.

Reply:
@tammybailey, what do you think?

Reply:
I prefer validator because it makes sense in an OO context. A validator is something that has actions and attributes. Validation is the process performed by the validator.

Reply:
Sounds good! Resolving this comment.